docs/source/significance.rst

```diff
@@ -0,0 +1,62 @@
+############
+Significance
+############
+
+How to access whether initialized ensembles are skillful? Is correlation of a lead
+timeseries different from zero? What is a probability that a retrospective forecast was
+more valueable than a historical simulation? How to correct for spatial autocorrelation?
+All these questions deal with significance. Here is how you can use ``climpred`` to
+calculate this. Please also have a look at the
+`significance example <examples/decadal/significance.html>`__.
+
+Metric-based p-value
+####################
+
+For the correlation `metrics <metrics.html>`__, like
+:py:func:`~climpred.metrics._pearson_r`, ``climpred`` also hosts the associated p-value, like
+:py:func:`~climpred.metrics._pearson_r_p_value`, that this correlation is significantly
+different from zero incorporating reduced degrees of freedom due to temporal
+autocorrelation.
+
+Bootstrapping with replacement
+##############################
+
```

Comment:
I'm trying to be careful with the use of the word "skill" since textbooks/definitions suggest that "skill" really only means that your initialized metric is better than some reference (e.g. persistence). I.e., a low MAE, high ACC, etc. can be skillful. Not just an ACC

Reply:
its really hard for me to not say skill... so rather just say metric? metric to me is the function and skill is just the value of that function.

Reply:
I completely agree with you here. I use "skill" a lot too and use it flexibly in the pH paper I just resubmitted. In the Jolliffe text and other forecasting papers I read that skill specifically means your initialized forecast beats some reference. I.e. you can't have "skill" without a reference. I would have to dig to find those lines again...

I just want to be careful with language. But I think skill can be used flexibly because it tends to be in the community. I think 'metric' makes the most sense here but if it sounds awkward you can use skill